File: UI/src/app/models/surtidos/duplicarSurtidoModel.ts
import { IOption } from 'ng-select';

export class DuplicarSurtidoModel {
    canal: Array<IOption>;
    cadenas: Array<IOption>;
    ensena: Array<IOption>;

    constructor() {
        this.canal = [];
        this.cadenas = [];
        this.ensena = [];
    }
}

File: Business/Model/Surtido/DuplicarSurtidoModel.cs
using Business.Model.common;
using System;
using System.Collections.Generic;
using System.Text;

namespace Business.Model.Surtido
{
    public class DuplicarSurtidoModel : BaseModel
    {
        public int idsurtido { get; set; }
        public int cliente { get; set; }
        public string canal { get; set; }
        public List<int> cadena { get; set; }
        public List<int> ensena { get; set; }
    }
}
